### PR TITLE
Polling exits when encountering an error

### PR DESCRIPTION
**Pull request by miketalley**
## Description and Context
Polling continues even when encountering an error

## Screenshots
image

## Who to Notify
@mattcoley @gcorne

### CODE REVIEW

File: packages/cli/lib/serverlessLogs.js [hunk 1]
```diff
@@ -63,6 +63,7 @@ const tailLogs = async ({
           })
         );
       }
+      process.exit();
```
Comment (gcorne) [inline packages/cli/lib/serverlessLogs.js:66]: Should we provide some feedback to the user that we have cancelled the polling?

Comment (miketalley) [reply packages/cli/lib/serverlessLogs.js:67]: I added a `logger.info` message "Stopped polling due to error", and the error should also be logged.

**Review by brandenrodgers — APPROVED** [review]
lgtm 👍

**Comment by mattcoley** [comment]
Any luck in merging this in? We got paged again for a similar issue.

**Pull request merged**